Firestore/core/src/firebase/firestore/local/leveldb_transaction.h

```diff
@@ -175,6 +178,15 @@ class LevelDbTransaction {
    */
   void Put(std::string key, std::string value);
 
+  /**
```

Comment:
Note: I think the Objective-C version of this function (above) had a bug: it's not delegating to the "regular" overload of `Put` and it forgets to remove the `key` from its `deletions_` step. I don't know if it could lead to actual incorrect behavior, though -- can you think of an example?

Reply:
The only way this would be problematic would be if we deleted/added a row in the same transaction, but I can't think of case where we would do that. Fixing the bug is definitely worthwhile.